401-code-challenges/Data-Structures/src/main/java/graph/Edge.java

```diff
@@ -0,0 +1,35 @@
+package graph;
+
+public class Edge {
+    private static final int DEFAULT_WEIGHT = 1;
+    Node source;
+    Node destination;
+    int weight;
+
+    Edge(Node source, Node destination){
+        this(source,destination,DEFAULT_WEIGHT);
+    }
+    Edge(Node src, Node dest, int weig){
```

Comment:
Everywhere else you call it `source` and then here you call is `src`, remember that consistency will make it easier for those reading your code (including yourself in the future!)